functions/api/transactions.js

```diff
@@ -90,7 +88,6 @@ router.get("/:id", isAdminOrSuperAdmin, async (request, response, next) => {
         // adding a day to the given query date as the new Date function returns 00:00:00 and we 
         // need to include the toDate for query .
         toDate.setDate(toDate.getDate()+1)
-        console.log(toDate);
```

Comment:
Is it suggested to remove all debug console statements in the api route functions ? or wud it be better to make it as console.debug

Reply:
i removed it because its just value without a msg. Thought it was added for testing. So, we want to have console logs with proper msg and some data if needed.